mender-convert-modify

```diff
@@ -219,4 +219,11 @@ platform_modify
 for overlay in "${overlays[@]}"; do
   log_info "Applying rootfs overlay: ${overlay}"
   run_and_log_cmd "sudo rsync --archive --keep-dirlinks --verbose ${overlay}/ work/rootfs/"
+
+  for append in $(cd work/rootfs; find . -name '*.mender-convert-append'); do
+      sourcefile=$(echo $append | sed 's@.mender-convert-append$@@')
+      log_info "Applying rootfs overlay append to: ${sourcefile}"
+      run_and_log_cmd "sudo cat work/rootfs/${append} >> work/rootfs/${sourcefile}"
+      run_and_log_cmd "sudo rm -f work/rootfs/${append}"
+  done
```

Comment:
Please reconsider.  We want this to be easily applied to other operating systems and as current projects have shown, people will want to use the demo server with this. I'm concerned that blindly overwriting the hosts file will break things in different OSes and especially if the end user application somehow uses the /etc/hosts file.

I am ok limiting this to the hosts file rather than '*.mender-convert-append' but it's hardly a complicated loop.

Reply:
We can keep it as-is, if you feel strongly about this. No need to limit it to `/etc/hosts`. 

A couple of comments anyway,

> but it's hardly a complicated loop

No I do not think that actual code is complicated, but it is additional method that we add to rootfs overlay which you can use to modify the target filesystem that needs to documented maintained, tested and so on. If this will ever be used to only modify `/etc/hosts`, imo it is not worth its own feature.

> blindly overwriting the hosts file will break things in different OSes and especially if the end user application somehow uses the /etc/hosts file.

As this is only relevant for the Demo Server, I do not see a big problem with this.

Reply:
OK.  I just removed it.